integration-tests/emulated/tests/assets/asset-hub-kusama/src/tests/mod.rs

```diff
@@ -26,3 +26,15 @@ emulated_integration_tests_common::include_penpal_create_foreign_asset_on_asset_
 	KUSAMA_ED,
 	system_parachains_constants::kusama::fee::WeightToFee
 );
+
+#[test]
+fn assets_can_be_claimed() {
```

Comment:
@franciscoaguirre @acatangiu 
We don't have tests directly in `mod.rs` files anywhere (polkadot-sdk, fellows), I don't know, probably ok and I am not against that. Just thinking,  we have also `set_xcm_versions.rs` that could be maybe change to `pallet_xcm.rs` or create `claim_assets.rs` or something like that.
Wdyt?

And also I would definitely suggest to create/add exactly the same tests to the `polkadot-sdk` repo, so:
- we can catch issues directly in polkadot-sdk (without need to patch/backport later)
- when we update intergration tests in the fellows, last time I compared diffs with polkadot-sdk, so ideally the only difference between fellows and polkadot-sdk tests is just wording/naming :) : `Rococo vs Kusama` or `Westend vs Polkadot` or `ROCs vs DOTs` ...

Reply:
I like the idea of putting it in a `pallet_xcm.rs` tests file. It's purpose would be to make sure pallet-xcm's extrinsics can be used in this runtime.

In polkadot-sdk we have unit tests for this extrinsic in pallet-xcm. I could add the integration tests to rococo and westend as well, just to make sure.

Reply:
Well, probably better to just make a new file `claim_assets.rs`. We could add tests that describe reasonable scenarios where assets would be trapped.

Reply:
> This starts me thinking that these claim_asset tests should also be maybe also moved directly to the runtimes as unit tests

Yes, the `xcm-emulator` is not really needed here, we just want to test that the runtime is properly configured to support `claim_assets` extrinsic.

Having said that, I would still keep it in xcm-emulator / emulated `claim_assets.rs` tests to start consolidating all XCM tests in single place.

I don't think it takes longer to run one vs the other so there's also no performance/speed loss/gains.